src/Microsoft.ServiceFabric.Powershell.Http/DeployMeshResourcesCmdlet.cs

```diff
@@ -0,0 +1,92 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.ServiceFabric.Powershell.Http
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Management.Automation;
+    using Microsoft.ServiceFabric.Client;
+
+    /// <summary>
+    /// Deploys mesh resources in a Service Fabric Mesh cluster.
+    /// </summary>
+    [Cmdlet("Deploy", "SFMeshResources")]
+    public class DeployMeshResourcesCmdlet : CommonCmdletBase
+    {
```

Comment:
As per email conversation between Raunak and Lubo, this cmdlet will verify if clsuter is running in mesh mode, is that not the case anymore, please confirm with Raunak.